server/methods/setUserPassword.js

```diff
@@ -0,0 +1,26 @@
+Meteor.methods({
+	setUserPassword(password) {
+		check(password, String);
+
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'setUserPassword'
+			});
+		}
+
+		const userId = Meteor.userId();
```

Comment:
Since you are setting this to avoid re-evaluating `Meteor.userId()`, you should probably move it to before line 5.